libbpf-rs/tests/bin/src/taskiter.bpf.c

```diff
@@ -0,0 +1,31 @@
+#include "vmlinux.h"
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+
+struct index_pid_pair {
+  uint32_t i;
+  pid_t pid;
+}
+
+uint32_t i = 0;
```

Comment:
use `static uint32_t i = 0;` to not export this type to userspace (`libbpf-cargo gen` will generate accessors for this otherwise)